images/tidb-operator-e2e/tidb-cluster-values.yaml

```diff
@@ -5,9 +5,12 @@
 rbac:
   create: true
 
-clusterName: demo-cluster
+# clusterName is the TiDB cluster name, it is required and should be unique
+# if multiple clusters are deployed in the same namespace.
+clusterName: demo
 
-# nodeSelectorRequired: true
+# timezone is the default system timzone for TiDB
+timezone: Asia/Shanghai
```

Comment:
Is this change supposed to be in this PR?

Reply:
No, it just keep this file sync with `charts/tidb-cluster/values.yaml`

Reply:
Its odd to pick a default. Can we use the timezone of the host machine or the helm user as the default?

Reply:
@tennix

Reply:
We can't get the host machine's timezone when installing cluster.

Reply:
Can we leave it empty and [require that it be set](https://github.com/helm/helm/blob/master/docs/charts_tips_and_tricks.md#know-your-template-functions)?

Reply:
The default value is just an example and users can change it when necessary just like the storage class field. I'm OK with other values or leave it empty as a required value.

Reply:
I think we need to assume that the values won't be changed unless explicitly called out either in the tutorial, or as a field in the MarketPlace, etc. I agree that if we do give it a value, UTC is best.